README.md

```diff
@@ -8,6 +8,16 @@ This application demonstrates a multi-party video application built with [twilio
 
 Run `npm install` to install all dependencies.
 
+The fastest way to get started is to use the Twilio CLI:
+
+- Install and configure the [Twilio CLI](https://www.twilio.com/docs/twilio-cli/quickstart).
+- Run `twilio plugins:install @twilio-labs/plugin-rtc` to install the WebRTC plugin.
```

Comment:
The name of the CLI plugin is `plugin-rtc`, which follows Twilio's convention for naming CLI plugins. I feel like 'server app plugin' could be confusing because it doesn't seem to relate to the name of the plugin that is installed. Should we improve the name of the plugin? Or is there something better that we can say here?

@aaalaniz What are your thoughts?